src/messaging/inbound/data-ingest/processors/comms.js

```diff
@@ -36,7 +35,6 @@ const processV2CommsData = async (message) => {
   }
 
   await persistCommsNotification(validated)
-  logger.info(`Comms message processed successfully, eventId: ${validated.id}`)
 }
```

Comment:
Moved the logging to be part of the saving to database component `repos/common/save-event.js`